src/Component/JsonSchema/Tests/Validation/ValidationTest.php

```diff
@@ -0,0 +1,631 @@
+<?php
+
+namespace Jane\Component\JsonSchema\Tests\Validation;
+
+use Jane\Component\JsonSchema\Console\Command\GenerateCommand;
+use Jane\Component\JsonSchema\Console\Loader\ConfigLoader;
+use Jane\Component\JsonSchema\Console\Loader\SchemaLoader;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Model\ArrayObject;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Model\FormatObject;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Model\NumericObject;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Model\ObjectObject;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Model\StringObject;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Normalizer\ArrayObjectNormalizer;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Normalizer\FormatObjectNormalizer;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Normalizer\NumericObjectNormalizer;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Normalizer\ObjectObjectNormalizer;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Normalizer\StringObjectNormalizer;
+use Jane\Component\JsonSchema\Tests\Validation\Generated\Validator\ValidationException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Validator\Constraints\Count;
+use Symfony\Component\Validator\Constraints\Hostname;
+
+class ValidationTest extends TestCase
+{
+    public function testValidation(): void
+    {
+        // 1. Generate
+        $command = new GenerateCommand(new ConfigLoader(), new SchemaLoader());
+        $inputArray = new ArrayInput([
+            '--config-file' => __DIR__ . \DIRECTORY_SEPARATOR . '.jane',
+        ], $command->getDefinition());
+
+        $command->execute($inputArray, new NullOutput());
+
+        // 2. Numeric --------------------------------------------------------------------------------------------------
+        $this->numericValidation();
+
+        // 3. String ---------------------------------------------------------------------------------------------------
+        $this->stringValidation();
+
+        // 4. Array ----------------------------------------------------------------------------------------------------
+        $this->arrayValidation();
+
+        // 5. Format ---------------------------------------------------------------------------------------------------
+        $this->formatValidation();
+
+        // 6. Object
+        $this->objectValidation();
+    }
+
+    private function numericValidation(): void
+    {
+        $normalizer = new NumericObjectNormalizer();
+
+        $data = $normalizer->denormalize([
```

Comment:
This is only testing `denormalize` process, I should add atleast one test with a `normalize` (even if it should work the same).